docs/node_scenarios.md

```diff
@@ -15,7 +15,18 @@ Following node chaos scenarios are supported:
 
 **NOTE**: node_start_scenario, node_stop_scenario, node_stop_start_scenario, node_termination_scenario, node_reboot_scenario and stop_start_kubelet_scenario are supported only on AWS as of now.
```

Comment:
Should have caught this before.  I think it is safe to say supported on AWS, GCP and Azure.   In the Azure commit you have AWS and Azure.   I will merge Azure first, but I am guessing we'll have some sort of merge conflict.

Reply:
Updated to say Aws, azure and gcp